assets/img/ogl.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Logo isn't very good quality

Reply:
SVG here: https://upload.wikimedia.org/wikipedia/en/4/46/UKOpenGovernmentLicence.svg

Reply:
Go with the better quality version.